flake.nix

```diff
@@ -13,16 +13,7 @@
 
     # keep-sorted start block=yes newline_separated=yes
     base16-fish = {
-      # Lock the base16-fish input to a custom patch [2] ("Make autosuggestion
-      # and comment base03"), since it is currently impossible to apply patches
-      # to flake inputs [1] ("Support flake references to patches").
-      #
-      # Once this single-patch approach no longer scales, the repository should
-      # be properly forked, if [2] has still not been merged.
```

Comment:
IMHO, it would be better to avoid a true fork in the sense that the repository owner changes. Instead you could open your patch as an upstream PR, and then I could maintain a semi-fork by opening an upstream PR that simply cherry picks our desired patches. Then, we simply update the `rev` of the repository to the cherry picked patchset. In other words, submit your PR upstream and ping me there, and I will get this sorted.

Also, submitting your patch upstream makes it more accessible to others.